README.md

```diff
@@ -1,24 +1,28 @@
 <p style="text-align:center;" align="center">
-   <picture align="center">
-      <source media="(prefers-color-scheme: dark)" srcset="./.github/assets/images/layer5/layer5-light-no-trim.svg">
-      <source media="(prefers-color-scheme: light)" srcset="./.github/assets/images/layer5/layer5-no-trim.svg">
-      <img align="center" src="./.github/assets/images/layer5/layer5-no-trim.svg" alt="Shows a dark layer5 logo in light mode and a white logo in dark mode" width="45%"/>
-   </picture>
+   <a href="https://github.com/layer5io/layer5">
+      <picture align="center">
+         <source media="(prefers-color-scheme: dark)" srcset="./.github/assets/images/layer5/layer5-light-no-trim.svg">
+         <source media="(prefers-color-scheme: light)" srcset="./.github/assets/images/layer5/layer5-no-trim.svg">
+         <img align="center" src="./.github/assets/images/layer5/layer5-no-trim.svg" alt="Shows a dark layer5 logo in light mode and a white logo in dark mode" width="45%"/>
+      </picture>
+   </a>
 </p>
 
 <p align="center">
 <a href="https://github.com/layer5io/layer5" alt="GitHub contributors">
 <img src="https://img.shields.io/github/contributors/layer5io/layer5.svg" /></a>
-<a href="https://github.com/layer5io/layer5" alt="GitHub issues by-label">
+<a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Alayer5labs+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+org%3A+label%3A%22help+wanted%22+" alt="Help wanted GitHub issues">
 <img src="https://img.shields.io/github/issues/layer5io/layer5/help%20wanted.svg?color=%23DDDD00" /></a>
```

Comment:
The img src also has to be updated to show the correct count of issues.

Reply:
It is showing correct no, of issues ref
![image](https://user-images.githubusercontent.com/95926324/194750369-d4dc6326-d53c-48ff-9c19-d335ffef4a05.png)

Reply:
Is it?
What's the count of issues you see when you click on the link? Isn't it "106"?
The count shouldn't be limited to this repo.

Reply:
Ah, I see I wonder if thats possible , will check , try and revert back 👍

Reply:
as expected no it wont show as per the url as our url is not a repo ,
and http://img.shield.io requires only 1 repo to show with the reqd label (I checked on official site) .

even meshery/meshery shows only 52 instead of 106 but it is correct as per meshery/meshery is concerned.
![image](https://user-images.githubusercontent.com/95926324/194751424-372d7c4a-2b1f-4692-a204-168668c6d626.png)

Reply:
Similarly It is showing here too @Nikhil-Ladha .

Reply:
Hmm 🤔 .... then, let it be as it is.